controllers/grid/pubIds/form/PKPAssignPublicIdentifiersForm.inc.php

```diff
@@ -66,6 +73,7 @@ function fetch($request) {
 		$templateMgr->assign('pubObject', $this->getPubObject());
 		$templateMgr->assign('approval', $this->getApproval());
 		$templateMgr->assign('confirmationText', $this->getConfirmationText());
+		$templateMgr->assign('formParams', $this->getFormParams());
```

Comment:
Alternately, is it possible to make some of the templates generic and share them in lib/pkp? It just seems clunky to have the variable provided only in the pkp-lib PHP code, and used only in the OJS template code.